src/components/CreateBaremetalHostDialog/CreateBaremetalHostDialog.js

```diff
@@ -17,6 +18,7 @@ const formFields = {
     id: 'controller',
     title: 'Management Controller Address',
     required: true,
+    validate: form => validateBmcURL(form.controller.value),
```

Comment:
you can use `settingsValue(form, 'controller')`

Reply:
That seems like an awfully specific (i.e. not general) function.

Reply:
it is actually meant to be used with `FormFactory` so it applies here too. Problem is that we need to change function name and name of the parameters and move it to better place to make it seem more generic :) I will refactor it.

Reply:
Done, I'm happy to update it once it's refactored.